### DISCUSSION TITLE
上海理工大学（申江校区/本部）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#27] [University]
> 回答来自群友

**位置情况**
- 有三个校区，分别在复兴路1195号、军工路1100号（申江校区）和军工路516号（即本部，580号和470号也有校门）
- 516号的校区分成南北两个校区，有天桥可以过去，或者从校外走
- 中英合作的项目在复兴路校区，机械大类、电子信息大类和环境大类的大一新生以及少数民族预科生第一年在申江校区，其余（如中德合作）在本部
- 复兴路校区挺靠近市中心的，步行十五分钟左右能到陕西南路地铁站。剩下两个相对比较偏但因为在杨浦所以还算在市里；申江校区骑车十五分钟能到地铁8号线延吉中路站；本部骑车二十分钟能到12号线爱国路站；这两个校区门口都有公交
没有去过复兴路校区，以下都是本部和申江校区的情况。

**上床下桌**
- 申江校区的女生宿舍都是上床下桌，男生大部分是上床下桌，小部分上下铺
- 本部男女生宿舍两种都有，具体住到哪种看运气（我们大一结束搬过去的时候是辅导员抽签），不知道大一就开始住的是怎么分的

**浴室**
- 卫生间里会有两个隔间，一个是蹲坑另一个什么都没有（可能是用于简单擦洗吧），不过本部的宿舍卫生间还是挺差的，申江校区要好一点；申江校区那间空的好像还有水龙头，但只有凉水
- 本部有一些宿舍会有澡堂（比如我宿舍的在一楼，但隔间很少），有些宿舍楼没有，没去过宿舍外面的澡堂不清楚距离；申江校区的澡堂离最远的女生宿舍都是三到五分钟步行可到

**晚自习**
- 大一有晚自习，在周一周三和周日，但如果有晚课就去上课；到了大二就没有了

**晨跑和运动打卡**
- 以前有晨跑次数要求现在无了，跑步+跳绳够次数就行
- 可以骑车

**寒暑假和小学期**
- 小学期算进在校时间，寒假大约一个月，暑假大约一个半月
- 小学期两个星期，有一些小学期课需要抢（比如金工实习），抢不到小学期的时间就只能在学期内上，会很占时间

**外卖**
- 允许点外卖
- 申江校区只能送到门口，不过校区本身就不大，从宿舍楼楼下到校门口可能也就三五分钟
- 我搬到本部的时候已经因为疫情封校了，外卖只能送门口，如果住得离校门口比较远（比如我寝）可能要走二十分钟

**洗衣机**
- 有
- 申江校区的新一点，比较干净；本部的可能比较旧和脏（包括洗衣房都很脏）
- 有些宿舍有阳台可以晾衣服，有些没有就只能晾在外面的架子上，一楼只能晾楼道里

**电/网**
- 网还可以
- 十一点到五点还是六点断电，同时断网（但不断空调）
- 期末季不断电
- 禁止大功率电器，不过每一层楼会有一个地方可以用吹风机

**带电脑**
- 能带，而且电子信息大类小学期是编程项目，还必须要用电脑

**Comment by WidthKim** [comment]
外卖补充
现在580 516 334门口都装上了外卖柜 甚至在四公三宿舍楼后面有一个很隐蔽的外卖柜叫春江路外卖柜可以点外卖

**Comment by BigSilverFish** [comment]
1100号现在工地很多，A/B区要绕很大一圈路到申二教和校门口（大概需要10分钟）
A1-A4宿舍被拆了，现在正在建1/2号公寓（14层楼）和3号公寓（9层楼），一个综合楼（17层）和一个五层教学楼，大概2026年6月30日竣工。25级的新生还要在工地待一年，但是26级的条件会好很多。
据说还有二期工程，会拆掉更多低层老建筑（A7，B区，C1，申阶梯等）
